benchmarks/util.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import freud
+
+
+def quatRandom():
```

Comment:
If you're already using rowan in a benchmark, you can probably use it here as well. I would probably remove this function entirely.